openscad.pro

```diff
@@ -1,3 +1,4 @@
+CONFIG+=experimental
```

Comment:
This shouldn't be the default

Reply:
Reverted, but I am kind of split on this. Yes, this should not/never come into the master branch, we agree on that.
On the other hand: This is a feature branch were we are working on an experimental feature.
Someone that is checking out an experimental feature branch may not know that on top of checking out the experimental branch has to explicitly add +CONFIG+=experimental
(even when knowing it, it is an extra step)
I guess the input-driver is kind of a special feature branch

Reply:
Good point. Perhaps @t-paul could comment on this as we need a mechanism for remembering to revert such things before merging to master.